src/components/ApplicationFooter.jsx

```diff
@@ -0,0 +1,167 @@
+    { text: "News and Info", link: "/news_and_info" },
+  ];
+
+  const navigationMenu = navigationItems.map((item) => (
+    <Link data-cy="link" to={item.link} style={styles.navLink}>
+      {item.text}
+    </Link>
+  ));
+
+  return (
+    <Box
+      data-cy="application-footer"
+      style={mobile ? styles.mobileFooter : styles.footer}
+    >
+      <Grid alignItems="center" justifyContent="center" container spacing={0}>
+        <Grid
+          data-cy="logo"
+          item
+          xs={12}
+          lg={3}
+          style={
+            mobile
+              ? { ...styles.mobileGridItem, ...borderBottom }
+              : { ...styles.gridItem, ...borderRight }
+          }
+        >
+          <img
+            style={styles.logo}
+            src={LogoCHWL}
+            alt="Community Health West London Logo"
+          />
+        </Grid>
+        <Grid
+          data-cy="about"
+          item
+          xs={12}
+          lg={3}
+          style={
+            mobile
+              ? { ...styles.mobileGridItem, ...borderBottom }
+              : { ...styles.gridItem, ...borderRight }
+          }
+        >
+          <Typography variant="body1" component="p" gutterBottom>
+            Community Health West London is a Community Interest Company made up
+            of six local charities. We are working together with the wider
+            community to improve the health and wellbeing of our residents.
+          </Typography>
+        </Grid>
+        <Grid
+          alignItems="center"
+          item
+          xs={12}
+          lg={3}
+          style={
+            mobile
+              ? { ...styles.mobileGridItem, ...borderBottom }
+              : { ...styles.gridItem, ...borderRight }
+          }
+        >
+          <Typography
+            data-cy="contacts"
+            variant="body1"
+            component="p"
+            gutterBottom
+            style={mobile ? styles.centerText : styles.longWord}
+          >
+            Phone: 0207 243 9806
+            <br />
+            info@communityhealthwestlondon.org.uk
+          </Typography>
+        </Grid>
+        <Grid
+          data-cy="navigation"
+          item
+          direction="column"
+          xs={12}
+          lg={3}
+          style={mobile ? styles.mobileGridItem : styles.gridItem}
+          alignItems="center"
+        >
+          <Box style={styles.navigation}>{navigationMenu}</Box>
+        </Grid>
+      </Grid>
+      <Typography
+        style={styles.centerText}
+        data-cy="disclamers"
+        variant="caption"
+        component="p"
+      >
+        This site is built according to Web Content Accessibility Guidlines
+        <br />
+        2020 All Rights Reserved by Community Health West London.
+      </Typography>
+    </Box>
+  );
+};
+
+export default ApplicationFooter;
+
+const borderBottom = { borderBottom: "1px solid rgba(0,0,0,0.3)" };
+const borderRight = { borderRight: "1px solid rgba(0,0,0,0.3)" };
```

Comment:
Why not have these two as part of `styles`?

Reply:
First we tried it and it didn't work so we did it as this because then it worked. But we must have missed something because i went back and checked again and it now worked in the style object.
So i added it in there and destructured  the style object to make the code a bite less messy.